terraform/environments/digital-prison-reporting/main.tf

```diff
@@ -998,6 +1035,49 @@ module "dynamo_tab_domain_registry" {
   )
 }
 
+# Dynamo table for StepFunctions DMS Tokens, DPR2-209
+module "dynamo_table_step_functions_token" {
```

Comment:
Dynamo table to cache the task token(s) for notifying step functions of the DMS task completion